yarn-project/aztec/src/cli/util.ts

```diff
@@ -56,7 +56,11 @@ export const parseModuleOptions = (options: string): Record<string, string> => {
   }
   const optionsArray = options.split(',');
   return optionsArray.reduce((acc, option) => {
-    const [key, value] = option.split('=');
+    const [key, value] = options.includes('=')
+      ? option.split('=')
+      : option.includes(':')
```

Comment:
Why do we need the colon variety?

Reply:
Honestly we don't but while testing I kept doing using colon instead of equals. i'll remove